scripts/generate-component-docs.rb

```diff
@@ -287,7 +287,9 @@ def get_docs_type_for_value(schema, value)
   if ['number', 'integer'].include?(value_type)
     numeric_type = get_schema_metadata(schema, 'docs::numeric_type')
     if numeric_type.nil?
-      @logger.error "All fields with numeric types should have 'docs::numeric_type' metadata included."
+      @logger.error "All fields with numeric types should have 'docs::numeric_type' metadata included." +
+        "e.g. #[configurable(metadata(docs::numeric_type = \"bytes\"))]"
```

Comment:
Does this get added on a newline? Should have its own `@logger.error`?

Reply:
The string on line 291 is part of the same line when sent to stdout. I don't have a strong preference, I just split it to not have long strings across the display.

Reply:
No real opinion either - figured I'd check cause I'm not super familiar with them rubies. I guess if the log line is _super long_ it may look funny but all's well.